### ISSUE TITLE
mysql_user module fails on database name with wildcard in grant string.

### ISSUE THREAD

**Issue by steverweber** [ansible/ansible#3957]
Ansible module mysql_user fails when including back-tics in the priv.
This issue prevents database names with wildcards.
http://stackoverflow.com/questions/2668591/grant-with-database-name-wildcard-in-mysql

```
  mysql_user: user=libtest host='%' password=pass priv='mysql.*:ALL/`lib\_%`.*:ALL,GRANT' state=present
```

The first pass completes with out error.  However on the second run.

```
def user_mod(cursor, user, host, password, new_priv):
... line: ~188 will fail because its trying to revoke with out the back tic ` :
... REVOKE ALL PRIVILEGES ON lib_%.* FROM 'lib'@'%' 
                privileges_revoke(cursor, user,host,db_table,grant_option)
...
```

The issue seems to be because privileges_get removes the backtick.

```
def privileges_get(cursor, user,host):

        db = res.group(2).replace('`', '')
...
```

so I changed this to:
        db = res.group(2)

This works however another issue comes into play because the module task never included back ticks for mysql:

```
REVOKE ALL PRIVILEGES ON `mysql`.* FROM 'libtest'@'%'
```

**Comment by steverweber** [comment]
In other words back ticks should not be removed from the privileges because they are required for wildcards.
`def privileges_get(cursor, user,host):`

**Comment by jimi-c** [comment]
I believe this was resolved by a recent commit (892c5943190fbd71d064649e36ea795536951394), as that also removed the line which removed the back-ticks. If not, let me know and we can re-open this.

Thanks!
